tests/e2e/createcollection.spec.ts

```diff
@@ -0,0 +1,67 @@
+import { expect, test } from '@playwright/test'
+
+const chains = ['Kusama', 'RMRK2', 'KusamaHub [Beta]', 'PolkadotHub [Beta]']
+
+test('Fill fields to check if they work', async ({ page }) => {
+  await page.goto('/create/collection')
+  //upload image logo
+  await page.setInputFiles('input[type="file"]', 'tests/e2e/unsplash-image.jpg')
+  //checks if Collection Name Title Exists
+  await expect(
+    page
+      .getByTestId('collection-name')
+      .filter({ has: page.getByText('Collection Name') })
+  ).toHaveCount(1)
+  await page
+    .getByTestId('collection-name')
+    .locator('input')
+    .fill('Best Collection Name')
+  // Fill collection Description
+  await page.getByTestId('collection-desc').fill('Best Collection Ever Created')
+  //active unlimited items switch
+  const collectionAmount = page.getByTestId('collection-maxAmount')
+  await collectionAmount.locator('.o-switch').click()
+  await collectionAmount.locator('.o-switch').click()
```

Comment:
two times click?

Reply:
ty, removed the second one and added a line to check if the related input shows up